libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -460,9 +462,9 @@ method publish*(g: GossipSub,
   trace "about to publish message on topic", name = topic,
                                              data = data.shortLog
 
-  var peers: HashSet[string]
   # TODO: we probably don't need to try multiple times
   if data.len > 0 and topic.len > 0:
+    var peers = g.mesh.getOrDefault(topic)
```

Comment:
re https://github.com/status-im/nim-libp2p/pull/243#issuecomment-650777922,
yeah, this seems correct, my concern was that we were pulling the node from the gossip table directly.